wolfcrypt/src/asn.c

```diff
@@ -24026,6 +24010,45 @@ static int SetReqAttrib(byte* output, char* pw, int pwPrintableString,
     return sz;
 }
 
+#ifdef WOLFSSL_CUSTOM_OID
+/* encode a custom oid and value */
+static int SetCustomObjectId(Cert* cert, byte* output, word32 outSz,
```

Comment:
Does this compile with ASN_TEMPLATE?

Reply:
This function `SetCustomObjectId` is used by both default ASN and template. All code is tested with both variants.